pkg/plugin/v1/init.go

```diff
@@ -139,15 +144,22 @@ func (p *initPlugin) Validate() error {
 	}
 
 	defaultOpts := chartutil.CreateOptions{CRDVersion: "v1"}
-	if !p.apiPlugin.gvk.Empty() || p.apiPlugin.createOptions != defaultOpts {
+	if !p.apiPlugin.apiFlags.gvk.Empty() || p.apiPlugin.apiFlags.createOptions != defaultOpts {
 		p.doAPIScaffold = true
-		return p.apiPlugin.Validate()
+		// should not do the create api validation because it has specifics checks that
+		// are not valid for the init plugin E.g ensure that the PROJECT/config file is scaffolded
+		return p.apiPlugin.apiFlags.Validate()
+	}
+
+	// set the repo to provide the operator name
+	if p.config.Repo == "" {
+		p.config.Repo = projectName
 	}
```

Comment:
the sdk olm features will look for the repo value. We could change that in the tool as well. But, would not make more sense set the repo and indeed allow users to inform it?

Reply:
It is the other option that I thought about